src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -162,7 +162,21 @@ const NetworkModificationNodeEditor = () => {
     const [messageId, setMessageId] = useState('');
     const [launchLoader, setLaunchLoader] = useState(false);
 
+    const CleanClipboard = () => {
+        setCopiedModifications([]);
+        snackInfo({
+            messageId: 'CopiedModificationInvalidationMessage',
+        });
```

Comment:
add translation of "CopiedModificationInvalidationMessage" in fr.json and en.json

Reply:
true, I didn't moved it from the #858 PR thanks !